pkg/proc/ppc64le_disasm.go

```diff
@@ -0,0 +1,162 @@
+package proc
+
+import (
+	"encoding/binary"
+
+	"github.com/go-delve/delve/pkg/dwarf/op"
+	"github.com/go-delve/delve/pkg/dwarf/regnum"
+	"golang.org/x/arch/ppc64/ppc64asm"
+)
+
+// Possible stacksplit prologues are inserted by stacksplit in
+// $GOROOT/src/cmd/internal/obj/ppc64/obj9.go.
+var prologuesPPC64LE []opcodeSeq
+
+func init() {
+	// Note: these will be the gnu opcodes and not the Go opcodes. Verify the sequences are as expected.
```

Comment:
TODO, also this is here but the comment in FirstPCAfterPrologue says they don't work.

Reply:
I still don't think these should be here unless the comment in FirstPCAfterPrologue is wrong.